webapp/components/navbar_dropdown.jsx

```diff
@@ -312,7 +309,7 @@ export default class NavbarDropdown extends React.Component {
                                 />
                             </a>
                         </li>
-                        {adminDivider}
```

Comment:
Ah that makes sense. It's a bit odd since you wouldn't be able to tell what's going on by just looking at the code (like how I needed it explained), but I think that's okay here.